crankers/src/metrics.rs

```diff
@@ -53,6 +53,17 @@ pub async fn emit_vault_metrics(
         })
         .count() as i64;
 
+    for (address, vault) in vaults.iter() {
+        datapoint_info!(
+            "restaking-vault-supply",
+            ("slot", slot as i64, i64),
+            ("slot_index", slot_index as i64, i64),
+            ("vault", address.to_string(), String),
```

Comment:
"vault" => address.to_string() so vault is a tag

Reply:
what would the full tuple look like then?

Reply:
i think you can do:
```
datapoint_info!(
            "restaking-vault-supply",
            "vault" => address.to_string(),
            ("slot", slot as i64, i64),
            ...);
```

Reply:
can do the same with vrt_mint too